src/config/store.ts

```diff
@@ -27,6 +28,7 @@ const isSerializable = (value: any) =>
   BigNumber.isBigNumber(value) || isPlain(value);
 
 const reducers = combineReducers({
+  aggregatedPools,
```

Comment:
open to naming ideas on this! Wanted to use the word "aggregated" to signify that it's not coming from Horizon, but from an aggregation API

Reply:
I like that name, pretty clear.